### ISSUE TITLE
Brackets

### ISSUE THREAD

**Issue by jael97** [adobe/brackets#14808]
My brackets has stopped working.... only the white space shows...... I can't even close it.... I don't know what the problem is... I uninstalled and installed it again but still doesn't work.... 
![Screenshot (2)](https://user-images.githubusercontent.com/51405551/58952751-68967800-87b6-11e9-8c9a-35b4c6f66159.png)

**Comment by weissjulian** [comment]
Yes, I know that problem. Open the Taskmanager and stop Brackets, than restart. Maybe you have to do this two or three times, if this doesn't work, reinstall it.
